plugin/documents.py

```diff
@@ -483,8 +483,17 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'region.yellowish lightbulb.lsp'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
-            suffix = 's' if action_count > 1 else ''
-            code_actions_link = make_command_link('lsp_code_actions', '{} code action{}'.format(action_count, suffix))
+            if action_count > 1:
+                code_actions_link = make_command_link('lsp_code_actions', '{} code actions'.format(action_count))
+            else:
+                for name, commands in responses.items():
+                    command = commands[0].get('command')
+                code_actions_link = make_command_link('lsp_execute', commands[0].get('title'), {
```

Comment:
This looks incomplete: the response type of a textDocument/codeAction request is

```
(Command | CodeAction)[] | null
```

you seem to be only accounting for type `CodeAction[]`, and then only when it has a `command` key. It could also have an `edit` key.

https://microsoft.github.io/language-server-protocol/specification#textDocument_codeAction

Reply:
By the way, to run any kind of code action, use `session.run_code_action`.

Reply:
> By the way, to run any kind of code action, use `session.run_code_action_async`.

Thanks for tips, I have updated in 968ef53